src/Service.Host/client/src/components/RootProductSearch.js

```diff
@@ -1,5 +1,5 @@
 ﻿import React, { Component } from 'react';
-import { FormGroup, FormControl, ControlLabel, Button, Modal, ListGroup, DropdownButton, Dropdown, MenuItem, InputGroup, ListGroupItem, Label, OverlayTrigger, Popover, Checkbox, Alert } from 'react-bootstrap';
+import { FormGroup, Glyphicon, FormControl, ControlLabel, Button, Modal, ListGroup, DropdownButton, Dropdown, MenuItem, InputGroup, ListGroupItem, Label, OverlayTrigger, Popover, Checkbox, Alert } from 'react-bootstrap';
```

Comment:
does Glyphicon actually get used - I can't see it but I may be missing something